README.rst

```diff
@@ -6,26 +6,14 @@
     :target: https://circleci.com/gh/dwavesystems/dwave-hybrid
     :alt: Linux/Mac build status
 
-.. image:: https://ci.appveyor.com/api/projects/status/porqyytww2elwjv8/branch/master?svg=true
-    :target: https://ci.appveyor.com/project/dwave-adtt/dwave-hybrid/branch/master
-    :alt: Windows build status
-
 .. image:: https://img.shields.io/codecov/c/github/dwavesystems/dwave-hybrid/master.svg
     :target: https://codecov.io/gh/dwavesystems/dwave-hybrid
     :alt: Code coverage
 
-.. image:: https://readthedocs.com/projects/d-wave-systems-dwave-hybrid/badge/?version=latest
-    :target: https://docs.ocean.dwavesys.com/projects/hybrid/en/latest/
-    :alt: Documentation status
-
 .. image:: https://img.shields.io/pypi/pyversions/dwave-hybrid.svg?style=flat
     :target: https://pypi.org/project/dwave-hybrid/
     :alt: PyPI - Python Version
 
-.. image:: https://cdn-assets.cloud.dwavesys.com/shared/latest/badges/leapide.svg
-    :target: https://ide.dwavesys.io/from-referrer
-    :alt: Open in Leap IDE
```

Comment:
Is the plan to add an open in leap for codespaces badge at some point?

Reply:
No such plan. The badge here served more as an example of adding the IDE badge to a ReST file. For Codespaces, we would also need to add the `.devcontainer` spec here. I doubt it would see much use.